src/main/java/frc/trigon/robot/subsystems/arm/armConstants.java

```diff
@@ -0,0 +1,65 @@
+package frc.trigon.robot.subsystems.arm;
+
+import com.ctre.phoenix.motorcontrol.NeutralMode;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import com.ctre.phoenix.sensors.AbsoluteSensorRange;
+import com.ctre.phoenix.sensors.CANCoder;
+import com.ctre.phoenix.sensors.CANCoderConfiguration;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel;
+import edu.wpi.first.math.controller.PIDController;
+
+public class armConstants {
+
+    private static final int
+            ANGLE_CAN_SPARK_ID = 1,
+            ELEVATOR_CAN_SPARK_ID = 2,
+            ANGLE_NEO_ID = 3,
+            ELEVATOR_NEO_ID = 4,
+            TALON_SRX_ID = 5,
+            CAN_CODER_ID = 6;
```

Comment:
Rework all the names here. For each part, you have a master motor and a follower motor. Never ever ever say `CAN_SPARK` or `NEO`. Say something like `ANGLE_MOTOR` etc.
Same goes for the encoders. Name them something like `ANGLE_ENCODER` and `ELEVATOR_ENCODER`

Reply:
what do I rename the neo to?

Reply:
At the end of the day, there should be no real mention to the neo here. It's a motor, that all. You have 4 motors, 2 for the angle and 2 for the elevator. For every pair, name one of them `MASTER` and one of them `FOLLOWER`

Reply:
so I should make two more CanSpark's and have one of them be a follower and the other be a master? or do I just remove the neo's and not replace them?

Reply:
What is the difference between a neo and a sparkmax?

Reply:
idk

Reply:
A spark max is a motor controller (בקר מנוע). A motor controller is what we use the connect to the motors and activate them. So a spark max is what controls the motor it's connected to. This time, we connected a neo motor to it, but in code, if we want to access it, we'll reference its motor controller.

Reply:
so should I make a MotorControllerGroup? like I did when I made the differential drive?

Reply:
No, just create both and configure them together.